### PR TITLE
Toggle info

### PR DESCRIPTION
**Pull request by AlexJanow**
Added Accordion for toggling info of SingleExercise description and variations with materialUI

### CODE REVIEW

File: src/pages/SingleExercise.css [hunk 1]
```diff
@@ -1,9 +1,12 @@
 .singleExercise__wrapper {
-  display: grid;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   background-color: var(--primaryBackground);
   color: var(--primary);
   grid-template-columns: 1fr;
   grid-template-rows: 1fr;
```
Comment (hellodebs) [inline src/pages/SingleExercise.css:8]: This has to be removed right?

File: src/pages/SingleExercise.js [hunk 2]
```diff
@@ -30,23 +48,56 @@ export default function SingleExercise() {
...
-                <li className="singleExercise__li" key={index}>
-                  <Link className="Link" to={`/training/${exercise}`}>
-                    {exercise}
-                  </Link>
-                </li>
-              );
-            })}
-        </ol>
+      <div className="singleExercise__accordion">
+        <Box display="flex" justifyContent="center" alignItems="center">
```
Comment (hellodebs) [inline src/pages/SingleExercise.js:52]: oh, this works?! Wouldn't have thought of that. But there are always many ways to one goal, right ;D

File: src/pages/SingleExercise.js [hunk 3]
```diff
@@ -30,23 +48,56 @@ export default function SingleExercise() {
...
+          <Accordion className={classes.root}>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              aria-controls="panel1a-content"
+              id="panel1a-header"
+            >
+              {" "}
```
Comment (hellodebs) [inline src/pages/SingleExercise.js:59]: prettier extension always doing it's job huh ;)

**Review by hellodebs — COMMENTED** [review]
Hey Alex, thank you I've learned from you. :) Good job!

File: src/pages/SingleExercise.js (hunk 2 above)
Comment (AlexJanow) [reply src/pages/SingleExercise.js:52]: Yeah for Material UI the box was the only solution that worked for me, tried a lot before doing it like this haha

File: src/pages/SingleExercise.js [hunk 4]
```diff
@@ -2,8 +2,26 @@ import React from "react";
...
+import Typography from "@material-ui/core/Typography";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { makeStyles } from "@material-ui/core/styles";
+import Box from "@material-ui/core/Box";
+
 import "./SingleExercise.css";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "80vw",
```
Comment (hugh-burgess) [inline src/pages/SingleExercise.js:17]: I’ve always taken `vw` and `vh` With a pinch of salt, as I’m pretty sure they ignore other rules in the code. Maybe try another measurement type?

File: src/pages/SingleExercise.js [hunk 5]
```diff
@@ -30,23 +48,56 @@ export default function SingleExercise() {
...
+              expandIcon={<ExpandMoreIcon />}
+              aria-controls="panel1a-content"
+              id="panel1a-header"
+            >
+              {" "}
+              <Typography className={classes.heading}>Description</Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <Typography>
+                {/* <div className="singleExercise__description"> */}
```
Comment (hugh-burgess) [inline src/pages/SingleExercise.js:64]: Does this need to be here?

File: src/pages/SingleExercise.js [hunk 6]
```diff
@@ -30,23 +48,56 @@ export default function SingleExercise() {
...
+        </Box>
+
+        <Box display="flex" justifyContent="center" alignItems="center">
+          <Accordion className={classes.root}>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              aria-controls="panel1a-content"
+              id="panel1a-header"
+            >
+              {" "}
```
Comment (hugh-burgess) [inline src/pages/SingleExercise.js:80]: This too?

**Review by hugh-burgess — COMMENTED** [review]
Hey mate great work. Couple things if you’re interested.. keep it up! 🏋️‍♂️ 💪

File: src/pages/SingleExercise.js (hunk 6 above)
Comment (AlexJanow) [reply src/pages/SingleExercise.js:80]: prettier extension is doing its best haha! Thanks mate, appreciate your time :)

File: src/pages/SingleExercise.js (hunk 4 above)
Comment (AlexJanow) [reply src/pages/SingleExercise.js:17]: which do you reccommend? Is % any better?

**Pull request merged**